index.js

```diff
@@ -166,6 +167,10 @@ module.exports = (md, options) => {
         md.use(require('./rules/embeds/compare'), safeObject(optsObj.compare));
     }
 
+    if (optsObj.collapsible_headings) {
+        md.use(require('./rules/collapsible_heading'), safeObject(optsObj.collapsible_headings));
+    }
```

Comment:
This is not an embed, should be on line 95

Reply:
Moved.